editor.planx.uk/src/@planx/components/Send/Editor.tsx

```diff
@@ -90,15 +90,7 @@ const SendComponent: React.FC<Props> = (props) => {
       //   Don't actually restrict selection because flowSlug matching is imperfect for some valid test cases
       const teamSlug = window.location.pathname?.split("/")?.[1];
       const flowSlug = window.location.pathname?.split("/")?.[2];
-      if (
-        value === Destination.BOPS &&
-        newCheckedValues.includes(value) &&
-        ![
-          "apply-for-a-lawful-development-certificate",
-          "apply-for-prior-approval",
-          "apply-for-planning-permission",
-        ].includes(flowSlug)
-      ) {
```

Comment:
Small unrelated change here: based on recent BOPS demos at Show & Tells, they can accept any ODP Schema application type & officers can self-setup in BOPS to receive it - so we don't need to warn about this !

Reply:
Nice to drop some of these hardcoded service names 👍